src/net/sourceforge/kolmafia/textui/RuntimeLibrary.java

```diff
@@ -5518,15 +5520,18 @@ public static Value closet_amount(ScriptRuntime controller, final Value arg) {
 
   public static Value equipped_amount(ScriptRuntime controller, final Value arg) {
     AdventureResult item = ItemPool.get((int) arg.intValue(), 0);
-    int runningTotal = 0;
+    int amount = InventoryManager.getEquippedCount(item);
```

Comment:
Both flavors of getEquippedCount are public and static.  I could make a case that calling the two parameter version here is clearer, especially in an IDE that will helpfully show that the parameter is named includeAllFamiliars.  I might go farther and wonder whether the one parameter version should be deleted and all calls to it add false as a second parameter.   I'm not certain some kind of parallelism with available_amount is beneficial but that is my opinion.  I checked and there are about 10 calls to the one parameter version.  So that change might be too intrusive.  This is not intended as a blocker.

Reply:
I think it's a reasonable case to make. There are a few cases in the codebase where searching all familiars would be potentially desired (if a bit slower than not searching).

For example, there's a "do we have Thor's Pliers" check that checks equipped items. We still have pliers even if they're on a Disembodied Hand we can't use. All these cases are fringe, though.